tests/bulk_create/tests.py

```diff
@@ -103,12 +103,12 @@ def test_non_auto_increment_pk_efficiency(self):
     @skipIfDBFeature('allows_auto_pk_0')
     def test_zero_as_autoval(self):
         """
-        Zero as id for AutoField should raise exception in MySQL, because MySQL
+        Zero as id for BigAutoField should raise exception in MySQL, because MySQL
         does not allow zero for automatic primary key.
         """
         valid_country = Country(name='Germany', iso_two_letter='DE')
         invalid_country = Country(id=0, name='Poland', iso_two_letter='PL')
-        msg = 'The database backend does not accept 0 as a value for AutoField.'
+        msg = 'The database backend does not accept 0 as a value for BigAutoField.'
```

Comment:
Analogous change in the `DatabaseOperations.validate_autopk_value()` should solve MySQL test failure.